prometheus/grr.libsonnet

```diff
@@ -0,0 +1,56 @@
+/*
+ * The `grr` function below will render prometheus resources generated by this
+ * prometheus library (e.g. in either `prometheusRules` or `prometheusAlerts`)
+ * into Kubernetes style objects, with kind/apiVersion. (Grizzly)[https://github.com/grafana/grizzly)
+ * has the ability to consume resources in this format.
```

Comment:
With above, this should be in one library, too much code duplication and grr.libsonnet is not relevant for deploying prometheus.

Reply:
the prometheus library isn't just about deploying prometheus, it also covers configuring it with rules/alerts/etc. This is relevant to that portion.